api/swagger.yaml

```
@@ -9709,6 +9723,82 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
       tags: ["System"]
+  # Using a zero-width character here to define a separate /_ping endpoint to describe the
+  # /_ping?capabilities=v1 API since otherwise Swagger won't allow us to define multiple
+  # responses for the same endpoint depending on query parameter.
+  # This is however supported by OpenAPI v3.
+  # See: https://github.com/OAI/OpenAPI-Specification/issues/146
+  # and https://github.com/OAI/OpenAPI-Specification/issues/182
+  /_ping​:
+    get:
+      summary: "/_ping?capabilities=v1"
```

Looks like we parse the argument now, so no longer allow a `v` prefix;

```suggestion
      summary: "/_ping?capabilities=1"
```